src/writer/proposal.ts

```diff
@@ -52,6 +58,21 @@ export async function verify(body): Promise<any> {
   } catch (e) {
     return Promise.reject('failed to check validation');
   }
+
+  try {
+    const [
+      { count_1d: proposalsDayCount, count_30d: proposalsMonthCount }
+    ] = await getRecentProposalsCount(space.id);
+
+    if (
+      proposalsDayCount >= proposalDayLimit ||
+      proposalsMonthCount >= proposalMonthLimit
+    ) {
+      return Promise.reject('too many proposals');
+    }
```

Comment:
What about like this?

```suggestion
    if (proposalsDayCount >= proposalDayLimit) {
      return Promise.reject('daily proposal limit reached');
    }
    if (proposalsMonthCount >= proposalMonthLimit) {
      return Promise.reject('monthly proposal limit reached');
    }
```

Reply:
@bonustrack What do you think?